include/ck/tensor_operation/gpu/device/device_conv2d_fwd_xdl_c_shuffle_nhwc_kyxc_nhwk.hpp

```diff
@@ -417,6 +417,8 @@ struct DeviceConv2dFwdXdl_C_Shuffle_Input_N_Hi_Wi_C_Weight_K_Y_X_C_Output_N_Ho_W
     using BGridDesc_K0_N_K1 = remove_cvref_t<decltype(ABCGridDescs{}[I1])>;
     using CGridDesc_M_N     = remove_cvref_t<decltype(ABCGridDescs{}[I2])>;
 
+    using Block2CTileMap = BlockToCTileMap_M00_N0_M01<MPerBlock, NPerBlock, CGridDesc_M_N>;
```

Comment:
@asroy Is there a performance concern not to use the default `BlockToCTileMap_M00_N0_M01Adapt` given by the underlying gridwise gemm? I can imagine for small problem sizes the overhead introduced by workgroup mapping can outweigh the benefits.

Reply:
For some reason, the case (in the PR description) run at 52TFlops with the ```BlockToCTileMap_M00_N0_M01``` with M01=1, and 44TFlops with ```BlockToCTileMap_M00_N0_M01Adapt```